cli/pack/deb.go

```diff
@@ -160,13 +160,8 @@ func createDebianBinary(packageDir string) error {
 }
 
 // getTntTTVersions checks the version of tarantool in bin_dir and returns it.
-func getTntTTVersions(packCtx *cmdcontext.PackCtx) (PackDependencies, error) {
-	tntVerBytes, err := exec.Command(filepath.Join(packCtx.App.BinDir, "tarantool"), "--version").
-		Output()
-	if err != nil {
-		return nil, err
-	}
-	tntVer, err := util.NormalizeGitVersion(string(tntVerBytes))
+func getTntTTVersions(cmdCtx *cmdcontext.CmdCtx) (PackDependencies, error) {
```

Comment:
If this function returns Tarantool version, why is "TT" in its name? Why Versions (plural form)?

Reply:
The name is correct at the moment, it returns tt version too now.

Reply:
Fixed.